lib/sensors/FSR.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "mbed.h"
+
+/*
+Class to read analog FSR values and convert them to digital ones
+*/
+
+class FSR{
+public:
+	FSR(PinName pin);
+
+	float getValue();
```

Comment:
Use units in the function name to make it clear what it is returning ex. `float getForce_Newtons()` or `float getForce_RawADC`